project/src/js/ui/clock/clockView.js

```diff
@@ -0,0 +1,68 @@
+import h from '../helper';
+import layout from '../layout';
+import i18n from '../../i18n';
+import clockSettings from './clockSettings';
+import { formatTimeinSecs } from '../../utils';
+import sound from '../../sound';
+
+export default function view(ctrl) {
+  window.StatusBar.hide(); // Put this here instead of ctrl so it is reapplied after a phone lock and unlock
```

Comment:
Neat. I didn't know there was as resume callback. I will change.

On Tue, Jun 21, 2016 at 9:48 AM, Vincent Velociter <notifications@github.com
> wrote:

> In project/src/js/ui/clock/clockView.js
> <https://github.com/veloce/lichobile/pull/354#discussion_r67870902>:
>
> > @@ -0,0 +1,68 @@
> > +import h from '../helper';
> > +import layout from '../layout';
> > +import i18n from '../../i18n';
> > +import clockSettings from './clockSettings';
> > +import { formatTimeinSecs } from '../../utils';
> > +import sound from '../../sound';
> > +
> > +export default function view(ctrl) {
> > +  window.StatusBar.hide(); // Put this here instead of ctrl so it is reapplied after a phone lock and unlock
>
> But here is it called on each redraw, which is not a good thing. You can
> simply listen to the cordova 'resume' event to hide again the status bar.
>
> You'll need to subscribe to the event and not forget to unsuscribe on
> unload so the behavior don't apply to other screens.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/veloce/lichobile/pull/354/files/47416a130f9a4a487049766a55dcb36b301e187e#r67870902>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA6RdqnJ13EebYNTj3sbjV3uGs2gmm9wks5qN-uggaJpZM4I06tl>
> .
>